src/components/notebook-utils.js

```diff
@@ -48,8 +48,18 @@ export const notebookNameValidator = existing => ({
     message: 'already exists'
   }
 })
+// removes leading dirs and a file ext suffix on paths
+export const getDisplayName = name => stripExtension(getFileName(name))
 
-export const notebookNameInput = ({ inputProps, ...props }) => h(ValidatedInput, {
+// removes all paths up to and including the last slash
+export const getFileName = _.flow(_.split('/'), _.last)
+
+// removes a file extension from a string
+export const stripExtension = _.replace(/\.[^/.]+$/, '')
+
+export const getExtension = _.flow(_.split('.'), _.last)
```

Comment:
hmm.. I think it would be, but there are only stub tests currently... I'll make a unit test ticket in the epic with this and runtime-utils as candidates for testing